lib/Forgotpassword.dart

```diff
@@ -0,0 +1,248 @@
+                            style: TextStyle(color: Colors.white),
+                            cursorColor: Colors.white,
+                            decoration: InputDecoration(
+                              fillColor: Color.fromRGBO(0, 0, 0, 0.3),
+                              filled: true,
+                              focusedBorder: OutlineInputBorder(
+                                borderRadius: BorderRadius.circular(20),
+                                borderSide: BorderSide(color: Colors.white),
+                              ),
+                              hintText: "New Password",
+                              hintStyle: TextStyle(color: Colors.white),
+                              border: OutlineInputBorder(
+                                borderRadius: BorderRadius.circular(20),
+                              ),
+                              prefixIcon: Icon(
+                                Icons.lock_outline,
+                                color: Colors.white,
+                              ),
+                              suffixIcon: IconButton(
+                                icon: eye == true
+                                    ? Icon(
+                                        Icons.remove_red_eye_outlined,
+                                        color: Colors.white,
+                                      )
+                                    : Icon(
+                                        Icons.remove_red_eye,
+                                        color: Colors.white,
+                                      ),
+                                onPressed: vaw,
+                              ),
+                            ),
+                          ),
+                        ),
+                        SizedBox(
+                          height: 8,
+                        ),
+                        SizedBox(
+                          height: 20,
+                        ),
+                        MaterialButton(
+                          color: Color.fromRGBO(0, 0, 0, 0.3),
+                          minWidth: MediaQuery.of(context).size.width * .8,
+                          height: 55,
+                          shape: RoundedRectangleBorder(
+                              borderRadius:
+                                  BorderRadius.all(Radius.circular(20.0))),
+                          onPressed: () {
+                            if (formKey.currentState!.validate()) {
+                              // Process data.
+                            }
+                          },
+                          child: Text(
+                            "Change Password",
+                            style: TextStyle(color: Colors.white),
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                ),
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+    // ),
```

Comment:
Never commit the commented code, and before raising a PR, you must review your code once. So in case you missed it while committing the code, You can catch it while raising a PR and updating it.